SmartDeviceLink/SDLTouchManager.m

```diff
@@ -257,16 +276,80 @@ - (void)sdl_handleTouchEnded:(SDLTouch *)touch {
                 self.singleTapTouch = nil;
             }
             break;
-        case SDLPerformingTouchTypeNone:
+
+            case SDLPerformingTouchTypeNone:
             break;
     }
     self.previousTouch = nil;
     _performingTouchType = SDLPerformingTouchTypeNone;
 }
 
+/**
+ *  Handles a CANCEL touch event sent by CORE. The CANCEL touch event is sent when a gesture is interrupted during a video stream. This can happen when a system dialog box appears on the screen, such as when the user is alerted about an incoming phone call. 
+ *
+ *  Pinch and pan gesture subscribers are notified if the gesture is canceled. Tap gestures are simply canceled without notification.
+ *
+ *  @param touch    Gesture information
+ */
+- (void)sdl_handleTouchCanceled:(SDLTouch *)touch {
+    if (!self.isTouchEnabled) {
```

Comment:
This already exists in the calling method and should not be repeated here or in the other sub-handlers.

Reply:
Fixed. Removed from all sub-handlers